tests/main.test.ts

```diff
@@ -27,14 +28,14 @@ test("test if env file value matched with actual value", async () => {
     if (actualMap.size != expectedMap.size) {
         console.log(`Expected: ${[...expectedMap.entries()]}`);
         console.log(`Actual: ${[...expectedMap.entries()]}`);
-        throw new Error("Size of two map doesn't matched");
+        throw new Error("Size of two map is not same");
     }
     for (const [key, val] of expectedMap) {
         const actualValue = actualMap.get(key);
         if (actualValue != val) {
-            console.log(`Expected: ${[...expectedMap.entries()]}`);
-            console.log(`Actual: ${[...expectedMap.entries()]}`);
-            throw new Error("Value doesn't matched");
+            console.log(`Expected: ${actualValue}`);
+            console.log(`Actual: ${val}`);
```

Comment:
These are backwards. Also the tests are failing, so the replace function must not be working.